.pre-commit-config.yaml

```diff
@@ -0,0 +1,13 @@
+repos:
+- repo: https://github.com/golangci/golangci-lint
+  rev: v1.55.2
+  hooks:
+    - id: golangci-lint # quickly check: only modified files are linted
+- repo: local
+  hooks:
+    - id: make-lint # use `make lint` for a full check
```

Comment:
We should only run of the diff of the committed changes otherwise this is way too long at every commit.